<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="192-168-1-4.tpgi.com.au" name="com.testingfoo.thanosair.test.BookFlightTest" tests="9" failures="0" timestamp="26 Jul 2019 00:49:01 GMT" time="20.878" errors="1">
  <testcase name="FlightDetailsTest" time="0.000" classname="com.testingfoo.thanosair.test.BookFlightTest">
    <skipped/>
  </testcase> <!-- FlightDetailsTest -->
  <testcase name="FlightOptionsTest" time="0.000" classname="com.testingfoo.thanosair.test.BookFlightTest">
    <skipped/>
  </testcase> <!-- FlightOptionsTest -->
  <testcase name="launchTest" time="3.439" classname="com.testingfoo.thanosair.test.BookFlightTest"/>
  <testcase name="flightSelectionTest" time="0.000" classname="com.testingfoo.thanosair.test.BookFlightTest">
    <skipped/>
  </testcase> <!-- flightSelectionTest -->
  <testcase name="launchTest" time="3.512" classname="com.testingfoo.thanosair.test.BookFlightTest"/>
  <testcase name="flightBooking" time="0.000" classname="com.testingfoo.thanosair.test.BookFlightTest">
    <skipped/>
  </testcase> <!-- flightBooking -->
  <testcase name="launchTest" time="6.829" classname="com.testingfoo.thanosair.test.BookFlightTest"/>
  <testcase name="launchTest" time="3.493" classname="com.testingfoo.thanosair.test.BookFlightTest">
    <error type="java.lang.NumberFormatException" message="For input string: &quot;4 &quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: "4 "
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
at java.lang.Integer.parseInt(Integer.java:580)
at java.lang.Integer.parseInt(Integer.java:615)
at com.testingfoo.thanosair.test.BookFlightTest.launchTest(BookFlightTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- launchTest -->
  <testcase name="launchTest" time="3.605" classname="com.testingfoo.thanosair.test.BookFlightTest"/>
</testsuite> <!-- com.testingfoo.thanosair.test.BookFlightTest -->
